src/axom/sidre/examples/spio/CMakeLists.txt

```diff
@@ -41,9 +41,15 @@ endforeach()
 
     if(AXOM_ENABLE_TESTS)
         if(SCR_FOUND)
-            axom_add_test(
-                NAME spio_IO_SCR_Checkpoint
-                COMMAND spio_IO_SCR_Checkpoint_ex )
+            if(ENABLE_MPI)
+                axom_add_test(
+                    NAME spio_IO_SCR_Checkpoint
+                    COMMAND spio_IO_SCR_Checkpoint_ex 
+                    NUM_MPI_TASKS 1 )
```

Comment:
Should the number of MPI ranks be bumped to a value > 1? @nselliott what do you think the value should be?

Reply:
Let's make it 2.